### DISCUSSION TITLE
Fibaro fgt001 / 0x010f reports additional battery level on endpoint 0

### DISCUSSION THREAD

**Discussion by kvaster** [zwave-js/zwave-js#6554] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

Fibaro thermostat fgt001 have special section for reportion endpoint 0 as endpoint 1 for battery level. This is cause endpoint 0 and endpoint 1 are duplicated and endpoint 2 reports battery level for external sensor. With recent version endpoint 1 and endpoint 2 are shown during introduce and are working OK. Manual refresh also shows values OK. But when battery level is reported due to changes (withour request) - it is reported on endpoint 0 and is not remapped.

Also battery values found during introduce are shown as 'Battery v1'. And battery values reported on endpoint 0 are shown as 'Battert v0' in zwavejs-ui.

### Software versions

Latest git as of this issue, but the problem was introduced since of September (approx).

### Device information

Fibaro fgt001

[zwave-1.tar.gz](https://github.com/zwave-js/node-zwave-js/files/13640296/zwave-1.tar.gz)
[zwave-2.tar.gz](https://github.com/zwave-js/node-zwave-js/files/13640298/zwave-2.tar.gz)

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [X] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

[a comment by an automated account is not shown]

**Comment by AlCalzone** [comment]
Looks like we already have special handling in place for this device, which maps the battery to endpoint 1, but it still exposes the value on endpoint 0.

**Comment by AlCalzone** [comment, marked as answer]
This is now tracked in https://github.com/zwave-js/node-zwave-js/issues/6555
